### ISSUE TITLE
Can't use built_value_generator: 8.0.0-nullsafety.0 because it depends on quiver <3.0.0

### ISSUE THREAD

**Issue by fabriziocacicia** [google/built_value.dart#976]
I wanted to use the nullsafety version of built_value_generator since I'm working on a project that already has all nullsafety dependencies.
But since built_value_generator 8.0.0-nullsafety.0 depends on quiver <3.0.0, I can't use it.

I'm sure that this would have been fixed, I just wanted to point it out.

**Comment by davidmorgan** [comment]
Sorry about that. Since it's a dev dependency, it should be possible to use a `dependency_override` on `quiver` to make it work:

```
dependency_overrides:
  quiver: ^3.0.0
```

It looks like quiver null safe release is still a prerelease, I'll wait for the real release to update.

**Comment by lukaszciastko** [comment]
I don't want to open a new thread for this, but built_value_generator is also throwing a dependency error when used with integration_test:

```
Because no versions of built_value_generator match >8.0.0 <9.0.0 and built_value_generator 8.0.0 depends on analyzer ^0.40.0, built_value_generator ^8.0.0 requires analyzer ^0.40.0.

And because analyzer >=0.39.5 <0.41.2 depends on crypto ^2.0.0 and every version of flutter_driver from sdk depends on crypto 3.0.0, built_value_generator ^8.0.0 is incompatible with flutter_driver from sdk.

So, because *** depends on both flutter_driver any from sdk and built_value_generator ^8.0.0, version solving failed.
pub get failed (1; So, because *** depends on both flutter_driver any from sdk and built_value_generator ^8.0.0, version solving failed.)
exit code 1
```

**Comment by davidmartos96** [comment]
@davidmorgan Experiencing the same issue with Flutter 2.0.0. It looks like version 3.0.0 or quiver is not a prerelease version anymore. https://pub.dev/packages/quiver

**Comment by davidmorgan** [comment]
I'd like to fix #984 before releasing a new version--if I just bumped deps then you would likely hit the runtime issue with null safety described in that issue. In https://github.com/google/built_value.dart/pull/1001 I fix that and a few other issues, hoping to get that reviewed, tested and released over the next few days.

Thanks everyone for your patience, and for using `built_value` :)

**Comment by davidmorgan** [comment]
This should now be fixed, `8.0.2` is released. Please let me know if you have further problems.
